src/platforms/hosted/cmsis_dap.c

```diff
@@ -292,7 +295,7 @@ static uint32_t dap_swd_dp_error(adiv5_debug_port_s *dp, const bool protocol_rec
 		 */
 		dap_line_reset();
 		if (dp->version >= 2)
-			dap_write_reg_no_check(dp, ADIV5_DP_TARGETSEL, dp->targetsel);
+			dap_dp_low_write(dp, ADIV5_DP_TARGETSEL, dp->targetsel);
```

Comment:
Given this change and that this is the only location dap_write_reg_no_check() is called (we couldn't find this variant of low-level write when making that change) please remove dap_write_reg_no_check() from the dap.h header and dap.c source file)

Reply:
done!